src/resources/js/commerce/gateway/paypal/checkout.js

```diff
@@ -358,9 +374,13 @@ tribe.tickets.commerce.gateway.paypal.checkout = {};
 	 * @return {void}
 	 */
 	obj.ready = function () {
-		$document.on( 'afterSetup.tribeTicketsCommerceCheckout', obj.setupButtons );
+		obj.setupLoader();
+
+		window.onload = (event) => {
+			obj.setupButtons( event, $( tribe.tickets.commerce.selectors.checkoutContainer ) );
+		};
```

Comment:
curious why this needs to be done on `onload` rather than just in the `ready` function?

Reply:
We are loading the external PayPal script using the template: https://github.com/the-events-calendar/event-tickets/blob/1a756c95dedbebcdb72bbd0b1e81302d7276382c/src/views/v2/commerce/gateway/paypal/checkout-script.php#L29-L28

So, the `ready` method triggers before PayPal is available causing an error.

Reply:
gotcha. could we maybe target that specific element for the `onload` event instead of the entire window?